app/controllers/api/v1/shared_accesses_controller.rb

```diff
@@ -44,8 +44,7 @@ def shareable_users
                               .where.not(id: [@room.shared_users.pluck(:id) << @room.user_id])
                               .where(role_id: [role_ids])
                               .search(params[:search])
-
-        render_data data: shareable_users, status: :ok
+        render_data data: shareable_users, status: :ok, serializer: SharedAccessSerializer
```

Comment:
Small comment - Swap the order of `status` and `seriliazer` params; for consistency throughout the app

Reply:
Besides that looks good! No need for rspec tests

Reply:
Agreed